documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1295,7 +1295,6 @@ sqlplus sys/top_secret@//localhost:1521/ORCLCDB as sysdba
 
   GRANT CREATE TABLE TO c##dbzuser CONTAINER=ALL;
   GRANT LOCK ANY TABLE TO c##dbzuser CONTAINER=ALL;
-  GRANT ALTER ANY TABLE TO c##dbzuser CONTAINER=ALL;
```

Comment:
While at it, do we actually need those CREATE grants for tables and sequences?

Reply:
Hi @gunnarmorling we do actually, the connector automatically creates the flush table if it doesn't already exist.